ui/package.json

```diff
@@ -29,22 +29,28 @@
     "react-dom": "16.8.6"
   },
   "devDependencies": {
-    "@connexta/ace": "git://github.com/connexta/ace.git#d48cdcd457289b66a310f2ab7ed4564ce5d8db93",
+    "@connexta/ace": "git://github.com/connexta/ace.git#235d5b3885e1525e2f42bfbd64cd9715d85f498c",
     "bootstrap-sass": "3.3.6",
     "bootswatch": "3.3.7",
     "compass-mixins": "0.12.10",
     "cpr": "3.0.1",
-    "lerna": "2.8.0",
+    "lerna": "3.16.4",
     "mkdirp": "0.5.1",
-    "node-sass": "4.9.4",
+    "node-sass": "4.12.0",
     "rimraf": "2.6.2",
     "sass-lint": "1.12.1"
   },
-  "workspaces": [
-    "packages/*"
-  ],
+  "workspaces": {
+    "packages": [
+      "packages/*"
+    ],
+    "nohoist": [
+      "**/styled-components"
+    ]
+  },
   "resolutions": {
     "admin-ui/@connexta/atlas/backbone.marionette": "1.8.8",
-    "@connexta/ace": "git://github.com/connexta/ace.git#d48cdcd457289b66a310f2ab7ed4564ce5d8db93"
-  }
+    "@connexta/ace": "git://github.com/connexta/ace.git#235d5b3885e1525e2f42bfbd64cd9715d85f498c"
```

Comment:
Why can't we do `*` here?

Reply:
We're trying to force a specific version.  I'm not sure you can use * in a resolutions declaration.